Project.toml

```diff
@@ -0,0 +1,15 @@
+name = "PyCall"
+uuid = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+authors = ["Steven G. Johnson <stevenj@mit.edu>", "Yichao Yu <yyc1992@gmail.com>", "Takafumi Arakaki <aka.tkf@gmail.com>", "Simon Kornblith <simon@simonster.com>", "Páll Haraldsson <Pall.Haraldsson@gmail.com>", "Jon Malmaud <malmaud@gmail.com>", "Jake Bolewski <jakebolewski@gmail.com>", "Keno Fischer <keno@alumni.harvard.edu>", "Joel Mason <jobba1@hotmail.com>", "Jameson Nash <vtjnash@gmail.com>", "The JuliaPy development team"]
```

Comment:
Not sure what to do here. List of authors are taken from top 10 `git log --format='%aN <%aE>' | sort | uniq -c | sort -nr`. I then appended `"The JuliaPy development team"`.

Reply:
that's fine, I guess